erpnext/public/js/controllers/transaction.js

```
@@ -473,6 +473,7 @@ erpnext.TransactionController = class TransactionController extends erpnext.taxe
 	}
 
 	refresh() {
+		debugger;
```

Biome (2.3.13)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)